textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/neextrator/NamedEntityExtractor.java

```diff
@@ -0,0 +1,92 @@
+package edu.uci.ics.textdb.dataflow.neextrator;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.common.exception.DataFlowException;
+
+import java.util.List;
+
+
+/**
+ * @author Feng [sam0227] on 4/27/16.
+ *
+ *         Wrap the Stanford NLP Named Entity Recognizer as an operator.
+ *         This operator would recognize 7 classes: Location, Person, Organization, Money, Percent, Date and Time.
+ *         Return the recoginized data as a list of spans.
+ *
+ *         For example: Given tuple with two field named: sentence1, sentence2.
+ *         tuple: ["Google is an organization.", "Its headquarter is in Mountain View."]
+ *         return:
+ *         ["sentence1,0,6,Google, ORGANIZATION", "sentence2,22,25,Mountain View, LOCATION"]
+ */
+
+public class NamedEntityExtractor implements IOperator {
+
+
+    private IOperator sourceOperator;
+    private ITuple sourceTuple;
+    private List<Attribute> searchInAttributes;
+
+
+    public static final String LOCATION = "Location";
```

Comment:
Since these names (e.g., "LOCATION" and "PERSON") are very general and can be used in other places, can we add an "NE_" prefix to each of them?  For instance, name them as "NE_LOCATION", "NE_PERSON", etc.